gslib/utils/hashing_helper.py

```diff
@@ -252,7 +262,7 @@ def CalculateMd5FromContents(fp):
 
 def Base64EncodeHash(digest_value):
   """Returns the base64-encoded version of the input hex digest value."""
```

Comment:
I'd expect something that returns a base64-encoded version of something to have a return type of `bytes`.  Can you add to the docstring, noting the return type is unicode?